ffi/build.py

```diff
@@ -205,7 +207,7 @@ def main_posix(kind, library_ext):
     except NotImplementedError:
         default_makeopts = ""
     makeopts = os.environ.get('LLVMLITE_MAKEOPTS', default_makeopts).split()
-    subprocess.check_call(['make', '-f', makefile] + makeopts)
+    subprocess.check_call(['make', '-f', makefile] + makeopts) #? should this also have `-B`?
```

Comment:
I think it probably should not - why might it be useful to have?

Reply:
I had issues when setting things up originally. It wouldn't recompile because nothing changed. When I added -B it recompiled regardless and then my environment started to work.

Reply:
OK - it sounds like we're in agreement that it's not necessary? (If so, this comment should be removed)